src/createTwitchVideoCompiler/index.js

```diff
@@ -81,7 +84,7 @@ export default function createTwitchVideoCompiler(
         eventManager.send({ name: 'downloadingClip', clip });
 
         const url = getTwitchClipVideoUrl(clip);
-        const downloadPath = path.join(outputDirectory, `./${clip.id}.mp4`);
+        const downloadPath = path.join(outputDirectory, `./clips/${clip.id}.mp4`);
```

Comment:
Don't you mean `const downloadPath = path.join(clipsDownloadPath, `./${clip.id}.mp4`);`?

Reply:
No, I wanted to store downloaded clips inside "clips" folder.
![image](https://user-images.githubusercontent.com/7059370/43991124-8d43eda0-9d6e-11e8-98c0-8187f1014176.png)